README.md

```diff
@@ -6,52 +6,64 @@ Try the current version at: [https://cs121-team-panda.github.io/coconut-interpre
 
 Developed by Jonathan Cruz, Teerapat Jenrungrot, Natalie Kadonaga, and Brittany Wang.
 
-## Installation
-```
-pip install -r requirements.txt
-```
+## Architecture 
+<img width="1093" alt="architecture" src="https://user-images.githubusercontent.com/35832643/38783767-b5ef5236-40bb-11e8-91b4-e1d5bdc0aa18.png">
 
-## Running the app
+The Coconut Online Interpreter project's architecture design is based on server-less computing based on AWS Lambda for executing user-submitted Coconut codes. The front-end of the project is developed using React. Based on the designed architecture shown above, the interpreter is resilient to any malicious code submitted by users since the backend protection is handled by AWS Lambda services.
+
+### Prerequisites
```

Comment:
This section should be moved after the Installation section and renamed to Running locally. This should take care of differentiating between AWS Lambda and Flask. `Requires Python 3.` can be removed (see my other comment for where it should be moved to).